code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -12,6 +12,7 @@
 /obj/item/explosive/grenade/frag/prime()
 	spawn(0)
 		explosion(loc, -1, -1, 3)
+		round_statistics.grenades_thrown++
```

Comment:
Add this to grenade/proc/activate() under the if(dangerous) check. It should cover all of this section, including the exception for training grenades.

Reply:
Simpler, I kinda wanted to exclude the smoke grenades, but I doubt it will be a real issue. good suggestion.